tests/cli_interactive.test

```diff
@@ -0,0 +1,104 @@
+# test for measuring the responsivity of interactive mode when given wrong input
+<
+f tests/test_short.txt
+a
+1
+1
+1
+1
+q
+$ ./spellcheck
+>
+------- Welcome to Spellcheck! ------
+
+Please load a file to begin. Additionally, select an output mode and/or choose dictionary before running the program.
+
+f [/path/file.txt]           : open file
+d [/path/dictionary.txt]     : input custom dictionary file
+m [number]                   : switch output mode - quiet (1) verbose (2) or interactive (3) 
+h                            : help
+q                            : quit program
+
+spellcheck > 
```

Comment:
You should try to input a file that doesn't open, a mode that doesn't exist, etc. More error cases need to be covered.
Also, the welcome to spellcheck sign is removed, and a color flag has been implemented.